src/main/java/net/mcreator/element/types/Attribute.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2024, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.element.types;
+
+import net.mcreator.element.GeneratableElement;
+import net.mcreator.element.parts.EntityEntry;
+import net.mcreator.workspace.elements.ModElement;
+
+import java.util.List;
+
+public class Attribute extends GeneratableElement {
+	public String name;

```

Comment:
After class declaration, we usually add new line in files in codebase. Currently this rule is not enforced by formatted, but we still to try to adhere to it. Applies to other files added in PR too